integration/js/app/video_test/video_test.html

```diff
@@ -62,6 +62,76 @@ <h2>Join a meeting</h2>
           Unbind Video Element
         </button>
       </form>
+
+       
+      <form id="add-video-tile-form">
+        <button id="add-video-tile-btn" type="submit">
+          addVideoTile
+        </button>
+        <label for="add-video-tile-label">Tile Id: </label>
+        <span id="video-tile-id"></span>
+      </form>
+
+      <form id="has-started-local-video-tile-form">
+        <button id="has-started-local-video-tile-btn" type="submit">
+          hasStartedLocalVideoTile
+        </button>
+        <label for="has-started-local-video-tile-label">Value: </label>
+        <span id="has-started-local-video-tile-boolean"></span>
+      </form>
+
+      <form id="get-local-video-tile-form">
+        <button id="get-local-video-tile-btn" type="submit">
+          getLocalVideoTile
+        </button>
+        <label for="local-video-tile-label">Tile Id: </label>
+        <span id="local-video-tile-id"></span>
+      </form>
+
+      <form id="get-all-video-tiles-form">
+        <button id="get-all-video-tiles-btn" type="submit">
+          getAllVideoTiles
+        </button>
+        <label for="total-video-tiles-label">Total Tiles Count: </label>
+        <span id="total-video-tiles-count"></span>
+      </form>
+
+      <form id="get-all-remote-video-tiles-form">
+        <button id="get-all-remote-video-tiles-btn" type="submit">
+          getAllRemoteVideoTiles
+        </button>
+        <label for="total-remote-video-tiles-label">Total Remote Tiles Count: </label>
+        <span id="total-remote-video-tiles-count"></span>
+      </form>
+
+      <form id="have-video-tile-for-attendeeId-form">
+        <button id="have-video-tile-for-attendeeId-btn" type="submit">
+          haveVideoTileForAttendeeId
```

Comment:
Is there an input where I can specify the attendee Id?

Reply:
The attendeeId would be the local attendeeId, which I get from the video_test.js to fetch from meeting session.

Reply:
But what if I want to check if there is a video tile for an attendee that is not the local attendee?